### PR TITLE
Display Workspaces in the output of pipeline/pipelinerun describe command

### LINKED ISSUE #1048: Pipeline/PipelineRun describe: display workspace
### Feature request

When doing a `tkn pipeline describe` (and `tkn pipelinerun describe`) I would like to see the declared workspace in the output.

### Use case

Being able to see what workspace  I need to provide to execute the pipeline.

### UI Example

```bash
$ tkn pipelinerun desc pipeline-release-run-z9557
Name:              pipeline-release-run-z9557
Namespace:         default
Pipeline Ref:          golang-test
Service Account:   release-right-meow
Timeout:           1h0m0s

# […]

📨 Workspace

 NAME       RESOURCE REF
 ∙ source   tekton-pipelines-git-v0-14-0

📡 Output Resources

 No output resources

⚓ Params

 NAME        VALUE
 ∙ package   github.com/tektoncd/pipeline
 ∙ flags     -v -mod=vendor


                  
```

### PR DESCRIPTION
**Pull request by vinamra28**
# Changes

The workspaces declared in the pipeline/pipelinerun will be displayed in the output of their respective describe command.

- When we will do `tkn pipeline desc <pipeline-name>` workspace name and description will be displayed in the output
- When we will do `tkn pipelinerun desc <pipelinerun-name>` workspace name, subdir and workspace binding will be displayed as part of output

Closes #1048 

Signed-off-by: vinamra28 <vinjain@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [X] Run the code checkers with `make check`
- [X] Regenerate the manpages, docs and go formatting with `make generated`
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes
```release-note
- Display workspace name and description in the output of pipeline describe command
- Display workspace name, subdir and workspace binding in the output of pipelinerun describe command
```

### CODE REVIEW

File: pkg/pipelinerun/description/description.go [hunk 1]
```diff
@@ -97,6 +97,21 @@ STARTED	DURATION	STATUS
...
 
+{{decorate "workspaces" ""}}{{decorate "underline bold" "Workspaces\n"}}
+
+{{- if eq (len .PipelineRun.Spec.Workspaces) 0 }}
+ No workspaces
+{{- else }}
+ NAME	SUB PATH		WORKSPACE BINDING
+{{- range $workspace := .PipelineRun.Spec.Workspaces }}
+{{- if not $workspace.SubPath }}
+ {{ decorate "bullet" $workspace.Name }}	{{ "---" }}		{{ formatWorkspace $workspace }}
```
Comment (piyush-garg) [inline pkg/pipelinerun/description/description.go:108]: I guess there are two tabs in between 2nd and 3rd column

Comment (vinamra28) [reply pkg/pipelinerun/description/description.go]: thanks for pointing out :+1: . Fixed

**Review by vdemeester — APPROVED** [review]
/meow
/cc @piyush-garg @pradeepitm12 @danielhelfand

File: pkg/pipelinerun/description/description.go [hunk 2]
```diff
@@ -97,6 +97,21 @@ STARTED	DURATION	STATUS
...
+
+{{- if eq (len .PipelineRun.Spec.Workspaces) 0 }}
+ No workspaces
+{{- else }}
+ NAME	SUB PATH	WORKSPACE BINDING
+{{- range $workspace := .PipelineRun.Spec.Workspaces }}
+{{- if not $workspace.SubPath }}
+ {{ decorate "bullet" $workspace.Name }}	{{ "---" }}	{{ formatWorkspace $workspace }}
+{{- else }}
+ {{ decorate "bullet" $workspace.Name }}	{{ $workspace.SubPath }} 	{{ formatWorkspace $workspace }}
```
Comment (piyush-garg) [inline pkg/pipelinerun/description/description.go:110]: It looks like more space than tab

**Review by piyush-garg — APPROVED** [review]
/lgtm

**Pull request merged**